esmvalcore/preprocessor/_multimodel.py

```diff
@@ -288,27 +297,47 @@ def _combine(cubes):
         concat_dim = iris.coords.AuxCoord(i, var_name=CONCAT_DIM)
         cube.add_aux_coord(concat_dim)
 
-    cubes = iris.cube.CubeList(cubes)
+    cubes = CubeList(cubes)
 
-    merged_cube = cubes.merge_cube()
+    try:
+        merged_cube = cubes.merge_cube()
+    except MergeError as exc:
+        raise ValueError(
+            "Multi-model statistics failed to merge input cubes into a single "
```

Comment:
```suggestion
            "Multi-model statistics failed to merge input cubes %s into a single " % cubes
```

Reply:
one thing here - maybe we should use `preprocessor._io.concatenate` here since that's very flexible with merging/concatenating vs the inbuilt iris function that's still very restrictive? Or do we want to be restrictive?

Reply:
I don't think that's possible/necessary:

1. We need `merge` here and not `concatenate`; `preprocessor._io` only provides `concatenate`.
2. AFAIK the main advantage of our custom function is that it can deal with duplicate/overlapping data .

Reply:
right! I had my doubts about that too so good to have them explained here, cheers, man!